pages/agent/v3/signed-pipelines.md

```diff
@@ -0,0 +1,114 @@
+# Signed pipelines
+
+## What are signed pipelines?
+
+Signed Pipelines is a security feature on the Buildkite Agent that allows you to cryptographically sign steps that are uploaded Buildkite, and then verify that signature on the Buildkite Agent before running the step. This can prevent a malicious actor from modifying the steps that are run on your agents, especially in the (hopefully unlikely) event that the Buildkite backend is compromised.
```

Comment:
> especially in the (hopefully unlikely) event that the Buildkite backend is compromised.

hmmmmmmmmm. Not sure the best way to talk about this. What this really allows is a zero-trust model. Perhaps something like "This allows you to assert that the steps the agent is being instructed to run were uploaded by your own agents, removing the necessity to trust Buildkite." ? Along those lines, not necessarily those words

Reply:
ooooh yesssssssss very good. will steal that